app/node_modules/dimensions/terrariaserverpackethandler.ts

```diff
@@ -174,7 +175,7 @@ class TerrariaServerPacketHandler {
           break;
       }
       client.sendChatMessage(message, color);
-      client.sendChatMessage("Reason: " + dcReasonText, color);
```

Comment:
> The hardcoded `Net.PlayerIsCreativeAndWorldIsNotCreative` translation was left untouched

The problem is that you left it in there but removed where it was used. It might be better to just remove it entirely if it's not going to be used.

Reply:
I left it in because the hardcoded message is different from the in-game one. The hardcoded one uses "dimension", while the in-game one uses "server": `Only Non-Journey characters may enter a Non-Journey server.`

I'll remove it if you favor the in-game one, if not then I'll correct it to use the hardcoded one for that case

Reply:
I think just use the in-game one, because that will be translated right?